src/monitors/Monitor.v3

```diff
@@ -40,3 +40,23 @@ class Monitor {
 	}
 }
 
+component MonitorUtil {
+	def probeOnEnterFunc(module: Module, func: FuncDecl, probe: FuncEnterProbe) {
+		var it = BytecodeIterator.new();
+		var bi = it.reset(func);
+		bi.current();
+		module.insertProbeAt(func.func_index, bi.pc, probe);
```

Comment:
I think we only want to use the `FrameAccessor` list trick if the first bytecode is `LOOP`, since it is more expensive than a regular probe (and creates garbage).